### PR TITLE
uppercase macAddress

### PR DESCRIPTION
**Pull request by alisonrag**

### CODE REVIEW

File: src/Network/Send.pm [hunk 1]
```diff
@@ -321,6 +321,7 @@ sub reconstruct_master_login {
 
 	$args->{ip} = '192.168.0.2' unless exists $args->{ip}; # gibberish
 	$args->{mac} = $config{macAddress} || '111111111111' unless exists $args->{mac}; # gibberish
```
Comment (ya4ept) [inline src/Network/Send.pm:323]: What about:
```perl
	$args->{mac} = uc($config{macAddress}) || '111111111111' unless exists $args->{mac}; # gibberish
```

**Review by ya4ept — CHANGES_REQUESTED** [review]

File: src/Network/Send.pm (hunk 1 above)
Comment (alisonrag) [reply src/Network/Send.pm:323]: i need to test, idk what perl return if we use uc in a not defined var

**Comment by alisonrag** [comment]
test:
```
use strict;
use warnings;

my $test;
my $output = uc($test) || 'test';
print $output."\n";
```

output:
```
$perl main.pl
test
Use of uninitialized value $test in uc at main.pl line 5.
```

this will still works, but will drop a warning in user console, what do you think? @ya4ept should we keep like it is?

**Comment by ya4ept** [comment]
```
$args->{mac} = uc($args->{mac});
```

**Comment by alisonrag** [comment]
what about now?

**Comment by ya4ept** [comment]
`unless exists $args->{mac};` it is not equivalent `if (exists $args->{mac})`
why did you decide to change it?

**Comment by alisonrag** [comment]
> `unless exists $args->{mac};` it is not equivalent `if (exists $args->{mac})`
> why did you decide to change it?

sorry, my mistake,  updated

**Review by ya4ept — APPROVED** [review]
looks nice

**Pull request merged**